### PR TITLE
feat: command describe domains

### PR DESCRIPTION
**Pull request by maxwelbm**
- command domains from describe

### CODE REVIEW

File: messages/describe/domains/errors.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package domains
+
+import "errors"
+
+var ErrorGetDomain = errors.New("Failed to your domain. Check your settings and try again. If the error persists, contact Azion support.")
```
Comment (helio-neto) [inline messages/describe/domains/errors.go:5]: The describe word was missing? just a doubt
```suggestion
var ErrorGetDomain = errors.New("Failed to describe your domain. Check your settings and try again. If the error persists, contact Azion support.")
```

Comment (PatrickMenoti) [reply messages/describe/domains/errors.go:5]: True. It's missing a word

File: pkg/cmd/describe/domains/fixtures/domain.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        "name": "Valideishoun",
+        "cnames": [],
+        "cname_access_only": false,
+        "digital_certificate_id": null,
+        "edge_application_id": 1674767911 ,
+        "is_active": true,
+        "domain_name": "oda1ssad1f1.map.azionedge.net"
+    },
+    "schema_version": 3
+}
```
Comment (PatrickMenoti) [inline pkg/cmd/describe/domains/fixtures/domain.json:13]: new line at the end of file

File: pkg/cmd/describe/domains/domains.go [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azion describe domains --domain-id 4312
+        $ azion describe domains --domain-id 1337 --out "./tmp/test.json" --format json
+        $ azion describe domains --domain-id 1337 --format json
+        `),
+		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (karane) [inline pkg/cmd/describe/domains/domains.go:36]: Why not define this function elsewhere and then, reference it in here?

Comment (maxwelbm) [reply pkg/cmd/describe/domains/domains.go:36]: We've already asked ourselves that! And what we see is that all the other CLI players follow the same line, so we prefer to leave it that way. And this is also something from cobra itself Examples, https://github.com/docker/cli/blob/master/cli/command/plugin/inspect.go, https://github.com/cli/cli/blob/trunk/pkg/cmd/gist/create/create.go

Comment (karane) [reply pkg/cmd/describe/domains/domains.go:36]: Very Ugly!

Comment (maxwelbm) [reply pkg/cmd/describe/domains/domains.go:36]: It would be great to receive suggestions for improvements @karane

**Review by PatrickMenoti — APPROVED** [review]

**Review by LucasGeneroso — APPROVED** [review]

**Pull request merged**